### PR TITLE
#1 added liquibase changesets for creation users table

### PR DESCRIPTION
**Pull request by dima-fedotovs**
this is very clear explanation about my change

### CODE REVIEW

**Comment by yekaterinasavelyeva** [comment]
good changes

File: src/main/resources/db/changeLog-0001-user.xml [hunk 1]
```diff
@@ -3,5 +3,30 @@
                    xmlns="http://www.liquibase.org/xml/ns/dbchangelog"
                    xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-3.5.xsd">
 
+    <changeSet id="create users table" author="Dimitrijs Fedotovs">
+        <createTable tableName="users">
+            <column name="id" type="uuid">
+                <constraints primaryKey="true" nullable="false"/>
```
Comment (dima-fedotovs) [inline src/main/resources/db/changeLog-0001-user.xml:9]: this is bad. Redo please!

**Review by yekaterinasavelyeva — APPROVED** [review]

File: src/main/resources/db/changeLog-0001-user.xml (hunk 1 above)
Comment (dima-fedotovs) [reply src/main/resources/db/changeLog-0001-user.xml:9]: I don't agree. It is pretty good one!

**Review by MaryR-code — APPROVED** [review]

**Review by Evgshul — APPROVED** [review]
Look's good

**Review by andrey-makeyev — APPROVED** [review]
nothing

**Pull request merged**